client/team-info.js

```diff
@@ -1,25 +1,27 @@
 module.exports = {
   'governance': [
-    { name: 'Mathias Buus', github: 'mafintosh', description: 'Mathias is a self taught JavaScript hacker from Copenhagen that has been working with Node.js since the 0.2 days. Mathias likes to work with P2P and distributed systems and is the author of more than 650 modules on npm. He is also the Chief of Research on the Beaker Browser, leading the technical work on the Dat Protocol.' },
-    { name: 'Karissa McKelvey', github: 'karissa', description: 'Karissa is a board member at Code for Science & Society and worked on the Dat Project since 2014. She is currently an open source developer at Digital Democracy. Former <a href="http://scholar.google.com/citations?user=RM2tB8EAAAAJ&hl=en">academic</a> experienced in building interactive data visualization and collaboration tools.' },
+    { name: 'Mathias Buus', github: 'mafintosh', description: 'Mathias is a self taught JavaScript hacker from Copenhagen that has been working with Node.js since the 0.2 days. Mathias likes to work with P2P and distributed systems and is the author of more than 650 modules on npm. He is a partner at <a href="https://github.com/hyperdivision">Hyperdivision</a>, leading the technical work on the Dat Protocol.' },
+    { name: 'Karissa McKelvey', github: 'karissa', description: 'Karissa is co-founded & is a board member at Code for Science & Society and worked on Dat since 2014. She is currently an open source developer at Digital Democracy. Former <a href="http://scholar.google.com/citations?user=RM2tB8EAAAAJ&hl=en">academic</a> experienced in building interactive data visualization and collaboration tools.' },
     { name: 'Danielle Robinson, PhD', github: 'daniellecrobinson', description: 'Danielle Robinson is Co-Executive Director of Code for Science & Society where she works support and advise people who create open source technology in the public interest. In 2016, she completed a PhD in neuroscience at Oregon Health & Science University. Then as a Mozilla Fellow for Science she ran Working Open Workshops, explored decentralized approaches to data preservation, and advocated for open access to research.' },
     { name: 'Joe Hand', github: 'joehand', description: 'Joe Hand is Co-Executive Director at Code for Science & Society and a core developer on Dat Project with a focus on developing and managing data-focused programs for researchers and community-driven organizations. Previously, Joe managed a global project at the Santa Fe Institute to transform data collection practices of an international NGO, operating in slums across 30 countries in the Global South' }
   ],
+  'other': [
+    {name: 'Georgiy Shibaev', github: 'RangerMauve', description: 'Based in Ottowa, Canada. Open source software engineer focused on developer experience, the command line tool, and documentation.'},
```

Comment:
Looks good to me! Only thing is that I can't legally call myself an Engineer in Canada because the laws are strict about that.

```suggestion
    {name: 'Georgiy Shibaev', github: 'RangerMauve', description: 'Based in Ottowa, Canada. Open source software developer focused on developer experience, the command line tool, and documentation.'},
```

Reply:
fascinating! developer it is then

Reply:
Basically, you need to go through this ethics training, have an engineering degree from University, and work under a certified engineer. It's supposed to mean that you have responsibility over the stuff you're making.

Pretty much the opposite of what tech people call engineers these days. :P